src/App.tsx

```diff
@@ -21,6 +22,7 @@ function App() {
           <Route path="/" element={<HomePage />} />
           <Route path="/login" element={<Login />} />
           <Route path="/create-liderbord" element={<CreateLiderbord />} />
+          <Route path="/search-liderbord" element={<SearchLiderbord />} />
```

Comment:
```suggestion
          <Route path="/search" element={<SearchLiderbord />} />
```

I think we can shorten the url path to just "search"since we can only search liderbords it's implied that when searching, you are searching for a liderbord, and shoter URLS are always nicer